ingest/ingestor.py

```diff
@@ -0,0 +1,19 @@
+import abc
+
+
+class Ingestor(metaclass=abc.ABCMeta):
+    """
+    The command interface that declares a method (execute) for a particular
+    action.
+    """
+    @abc.abstractmethod
+    def execute_ingest(self, ):
+        pass
+
+    """
+    Returns true/false for whether this ingestor can handle the given file type
+    type should correspond to the
+    """
+    @abc.abstractmethod
+    def matches_file_type(self, type):
+        pass
```

Comment:
This doc should be on the line after the function signature, so IDEs can auto-detect it.  Also, the doc itself looks truncated at the end ("...corresponds to the" what?).

```suggestion
    @abc.abstractmethod
    def matches_file_type(self, type):
        """Returns true/false for whether this ingestor can handle the given file type
        type should correspond to the
        """
        pass
```

Reply:
Via [pep 257 -- Docstring Conventions](https://www.python.org/dev/peps/pep-0257/)  under [Multi-line Doc string](https://www.python.org/dev/peps/pep-0257/#id17) "The summary line may be on the same line as the opening quotes or on the next line."

Reply:
The issue here is having the docstring above the function signature -- the issue isn't the position of the summary line within the multiline docstring.  Your quoted clause from PEP 257 means this would be OK (albeit less common):

```
    @abc.abstractmethod
    def matches_file_type(self, type):
        """
        Returns true/false for whether this ingestor can handle the given file type
        type should correspond to the
        """
```

But no examples in PEP 257 put the docstring _above_ the signature -- so the suggested change is more conventional.  

That said, my comment was more motivated more by functional effect on developer experience than style compliance. 
 As is, the docstring cannot be auto-detected by IDEs.  The suggested change fixes that DX paper-cut.